backend/package.json

```
@@ -28,6 +28,7 @@
   "scripts": {
     "serve": "cross-env NODE_ENV=development node server.js",
     "lint": "eslint server.js . --max-warnings=0",
+    "lint-sarif": "eslint server.js . --max-warnings=0 --format=@microsoft/eslint-formatter-sarif",
```

```suggestion
    "lint-sarif": "yarn lint --format=@microsoft/eslint-formatter-sarif",
```
Should be possible. Same for the other occurences